### DISCUSSION TITLE
The Fifth Element isn't shown in sliding menu (Avalonia.Skia.Svg)

### DISCUSSION THREAD

**Discussion by Yveske1966** [AvaloniaUI/Avalonia#14197] [Q&A] [answered]
Full sample program in attachment

In the sliding menu, the 5th svg isn't shown (for that manner, everything after the 4th isn't shown).
![The fifth element](https://github.com/AvaloniaUI/Avalonia/assets/68047793/85754869-6740-4411-b8b7-ab4486ea28b3)

I copied the content of the 5th element in the file of the 1st and now it is shown.
![Missing](https://github.com/AvaloniaUI/Avalonia/assets/68047793/fd9dbdfc-f1a8-49bf-9b99-70222bc9f230)

Source (Colors.xaml)
![Source](https://github.com/AvaloniaUI/Avalonia/assets/68047793/493b6f0e-c41c-4af8-9688-a0bd75d8c474)

[TheFifthElement.zip](https://github.com/AvaloniaUI/Avalonia/files/13931088/TheFifthElement.zip)

What's wrong here ?

**Comment by thevortexcloud** [comment]
Huh. That package just[ a few weeks ago](https://github.com/AvaloniaUI/Avalonia/discussions/13983) was not in the Avalonia namespace and was instead a third party one. I wonder when it moved... Or perhaps I just misremembering. In any case, the source code seems to be s[till hosted outside the Avalonia org](https://github.com/wieslawsoltes/Svg.Skia), So I am unclear exactly as to who is responsible for maintaining this. You might be better off asking over there in this case. There is this open issue which seems to indicate something broke in a recent version: https://github.com/wieslawsoltes/Svg.Skia/issues/200

**Reply by Yveske1966** [reply]
Made an entry on GitHub of Wieslawsoltes (https://github.com/wieslawsoltes/Svg.Skia/issues/201)

**Comment by timunie** [comment]
Check dev tools F12 if there is any limit width or clipping going on

**Comment by timunie** [comment, marked as answer]
SVG is a control and thus should not be used in a way you did it via DynamicResource. Either wrap it in a Template or (my favorite here) use the PathData with a PathIcon control. 

Anyways, you have an additional `"` in the svg source on the last resource. Please fix it. 

![image](https://github.com/AvaloniaUI/Avalonia/assets/47110241/edf512f5-b84b-44ee-883e-18432a59dfed)

**Reply by Yveske1966** [reply]
Facepalm ! That fixed it !
Many thanks !!!!
You don't want to know how many hours were lost by this and how many times I walked over the code not noticing that mistake...
